src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java

```diff
@@ -141,7 +141,9 @@ public static void rect(Fluid fluid, int left, int top, int width, int height, i
 	 * Draws a beveled, round rectangle that is substantially similar to default Minecraft UI panels.
 	 */
 	public static void drawGuiPanel(int x, int y, int width, int height) {
-		drawGuiPanel(x, y, width, height, 0x555555, 0xC6C6C6, 0xFFFFFF, 0x000000);
+		// For the purposes of Java's weird love-hate relationship with unsigned integers,
```

Comment:
Ultimately unnecessary: 0xFFFFFFFF is a valid int constant.

Reply:
See above: IntelliJ doesn't know _what_ it wants...fixed!